src/Kafka/KafkaInstanceStatus/components/SuspendedPopover1.tsx

```diff
@@ -0,0 +1,38 @@
+import { Popover, TextContent } from "@patternfly/react-core";
+import type { RefObject, VoidFunctionComponent } from "react";
+import { ExclamationTriangleIcon } from "@patternfly/react-icons";
+import { useTranslation } from "react-i18next";
+import { ExternalLink } from "../../../shared";
+import type { PopoverProps } from "@patternfly/react-core";
+
+export type SuspendedPopover1Props = {
+  children?: PopoverProps["children"];
+  reference?: RefObject<HTMLButtonElement>;
+};
+
+export const SuspendedPopover1: VoidFunctionComponent<
```

Comment:
Is this component meant to be here? I don't see it used anywhere. In case we do, can we please find a better name for this? Or at least add some comment that describes why we need it?

Reply:
Initially, I created a component with an external link but then I got to know that the external link is not yet ready so created another component with not no link.

Reply:
`SuspendedPopoverNoSupportLink` I think would be a better name then, don't you think?